### ISSUE TITLE
Need option to override default "tap closes" option on core-dropdown-base

### ISSUE THREAD

**Issue by wintersandroid** [Polymer/core-dropdown#12]
Core dropdown base implements a on-tap which toggles the overlay, this should be disabled by an attribute as if you have buttons or other tappable components in the overlay, the overlay closes..

I had to override with 
```
  ready: function(){
        this.$.menu_popup.toggleOverlay = function(){};
      },
```
where menu_popup is the core-dropdown-base

**Comment by W64BugBear** [comment]
that's not really a fix in case of a paper-dropdown-menu, because the dropdown wont open anymore...

**Comment by morethanreal** [comment]
Fixed in Polymer/core-dropdown@53abf8d
